modules/support-gather-data.adoc

```diff
@@ -38,6 +38,11 @@ If your cluster is using a restricted network, you must take additional steps. I
 $ oc import-image is/must-gather -n openshift
 ----
 ====
++
+[NOTE]
```

Comment:
By adding this NOTE here, there are now 3 admonitions in a row. Per the SSG [1], we should be avoiding putting admonitions back-to-back like this.

I'd suggest to take another look and see what else you could adjust so that we don't end up with 3 back-to-back. The SSG offers a suggestion where you could take the least important one and move it out of a note and put it in the main content. Of the 3 of them, your new one seems like the one that is most appropriate as a note. The other two could possibly be added as substeps to step 2. 

[1] https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
I have done quite a bit of changes as per your suggestion. Kindly review and let me know.

Reply:
@snarayan-redhat this looks much better IMO. Thank you so much for taking the time to move things around.

I think that keeping it as a note in step 1 is fine, so you can go ahead and delete your commented out steps. Thanks again!